.github/workflows/test-build.yml

```diff
@@ -9,5 +9,5 @@ jobs:
         with:
           node-version: 14
       - run: yarn install
-      # - run: yarn test
+      - run: yarn test:jest

```

Comment:
I would run both `yarn test` and `yarn test:jest` bc `yarn test` runs the existing tests in `./test`

Reply:
`yarn test` is broken right now. 😬 More refactoring might fix them.